content/authors/yichen-li/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Yichen Li
+role: Graduate Student
+avatar_filename: yichen-lab-photo.jpg
+bio: I am interested in using methods from cognitive science, neuroscience, and
+  computational modeling to understand the mechanisms behind common-sense
+  reasoning, the underlying representations of abstract concepts, and the
+  operations that build our compositional thoughts.
+interests:
+  - Intuitive Physics
+  - Object Representations
+  - Program Induction
+  - Compositional Thinking
+education:
+  courses:
+    - course: BA in Mathematics, Computer Science
+      institution: New York University
+      year: 2018
+    - course: PhD in Cognitive Science
+      institution: Harvard University
+      year: -1
+user_groups:
+  - Graduate Students
```

Comment:
Can you add "Researchers" as well – it's for the backend. Working on a way to have this default, but not there yet.